### PR TITLE
Feature/video

### PR DESCRIPTION
**Pull request by tomkrupka**
- Make a basic video component using a youtube url
- Make it optional to use a custom image or it will default to the youtube thumbnail

Using custom image
Screen Shot 2019-04-26 at 4 41 31 PM

No custom image using youtube thumbnail
Screen Shot 2019-04-26 at 4 41 50 PM

### CODE REVIEW

File: resources/scss/components/_icons.scss [hunk 1]
```diff
@@ -17,3 +17,11 @@
         transform: rotate(90deg);
     }
 }
+
+.right-triangle {
```
Comment (robertvrabel) [inline resources/scss/components/_icons.scss]: Is this needed even though it looks like an SVG was added: `svg/video-play.blade.php` ?

File: tailwind.js [hunk 2]
```diff
@@ -213,6 +213,8 @@ module.exports = {
         '3xl': '1.875rem', // 30px
         '4xl': '2.25rem', // 36px
         '5xl': '3rem', // 48px
+        '7xl': '4rem', // 64px
```
Comment (robertvrabel) [inline tailwind.js]: Can you confirm all these new additions are actually being used? I don't see any references to them in the file `resources/views/components/video.blade.php`. Maybe I'm missing something though!

Comment (tomkrupka) [reply tailwind.js]: I will confirm and double check thanks!!

**Comment by nickdenardis** [comment]
👍

**Comment by robertvrabel** [comment]
Looks like we are good to merge this in now!

**Comment by tomkrupka** [comment]
Thanks! This is now merged into development.

**Pull request closed without merging**